src/logo_downloader.py

```diff
@@ -161,7 +161,7 @@ def fetch_teams_data(self, league: str) -> Optional[Dict]:
         
         try:
             logger.info(f"Fetching team data for {league} from ESPN API...")
-            response = self.session.get(api_url, headers=self.headers, timeout=self.request_timeout)
+            response = self.session.get(api_url, params={'limit':1000},headers=self.headers, timeout=self.request_timeout)
```

Comment:
I did this because Teams I was using didn't load properly. It seems ESPN has a default limit in API calls, so 1000 seems to work. Same for the NFL, NCAA FB managers that i was using and it wasn't loading the future events without it.